satpy/tests/writer_tests/test_ninjotiff.py

```diff
@@ -82,3 +84,36 @@ def test_image(self, iwsi, save_dataset):
         assert(nt.save.mock_calls[0][2]['ch_min_measurement_unit']
                < nt.save.mock_calls[0][2]['ch_max_measurement_unit'])
         assert(ret == nt.save.return_value)
+
+    def test_convert_units(self):
```

Comment:
Could you make three test functions here instead?

Reply:
Done.